wp-content/themes/ipbs-largo/inc/enqueue.php

```diff
@@ -8,5 +8,14 @@ function ipbs_enqueue_styles(){
 		get_stylesheet_directory_uri() . '/css/child-style.css',
 		filemtime( get_stylesheet_directory() . '/css/child-style.css' )
 	);
+	// fonts
```

Comment:
Not a comment on this change specifically, but in the `'largo-child-styles'` enqueue above, `filemtime( ... )` should be the 4th param, with the third as `null`.

Reply:
Updated in https://github.com/INN/umbrella-ipbs/pull/65/commits/fa1eb13ee0eb3f3f29bb521238621e2927c9cee0